packages/fx-core/src/plugins/resource/localdebug/tasks.ts

```diff
@@ -122,6 +122,16 @@ export function generateSpfxTasks(): Record<string, unknown>[] {
       },
       dependsOn: "gulp trust-dev-cert",
```

Comment:
Since gulp has its own dev cert, does SPFx require local-debug plugin to setup teamsfx certificate as well?

Currently, command `fx-extension.pre-debug-check` generates teamsfx certificate no matter spfx or not.

Reply:
My suggestion is to try this new spfx local debug experience on a clean machine (e.g., windows sandbox) to see if any functionality is redundant (cert, deps checker, ...).